Language/Reference/User-Interface-Help/getfilename-method-visual-basic-for-applications.md

```diff
@@ -26,7 +26,7 @@ The **GetFileName** method syntax has these parts:
 
 ## Remarks
 
-The **GetFileName** method returns a zero-length string ("") if _pathspec_ does not end with the named component.
+The **GetFileName** method returns a zero-length string ("") if _pathspec_ contains a drive specification only (Example: "C:\"), otherwise it will return the last component in the path, even if that component is the name of a folder and not a file.
```

Comment:
```suggestion
The **GetFileName** method returns a zero-length string ("") if _pathspec_ contains a drive specification only (Example: "C:\"), otherwise it returns the last component in the path, even if that component is the name of a folder and not a file.
```

Reply:
Match tense with previous "method returns". Thanks for the fix